examples/Vanilla/OnLoad/onload.js

```diff
@@ -5,8 +5,22 @@ function rciMainAction(tenancyId, rciSdk) {
 
   // Step 2: Capture your default collectors
   let defaults = rciSdk.collector.defaultCollectors;
+
+  /**
+   * @type {Config}
+   */
   const config = {
-    actions: true
+    actions: true,
+    actionsBatching: false,
+    events: [
+      {
+        selector: 'window', eventName: 'load', eventCategory: 'Document', scope: 'state'
```

Comment:
@robertdumitrescu CONSISTENCY - The naming conventions here are becoming a little unclear - particularly as the names crossover from Web APIs to RCI's Data API.  I would suggest referencing terms from the [Moz documentation](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener) and the [Data API ](https://docs.real-user-data.eggplant.cloud/open-api/index.html) directly, for example:

```
{
        target: 'window', type: 'load', rciEventAction: 'full_page_load', rciEventType: 'state'
}
```

I would also suggest that we plumb the `rciEventAction` value all the way through to `eventAction` in the Data API.